tests/sles4sap/hana.pm

```diff
@@ -45,9 +46,15 @@ sub run {
     # Add host's IP to /etc/hosts
     select_console 'root-console';
     assert_script_run 'echo $(ip -4 addr show dev eth0 | sed -rne "/inet/s/[[:blank:]]*inet ([0-9\.]*).*/\1/p") $(hostname) >> /etc/hosts';
+    if (is_sle('>=15')) {
+        my $arch       = get_required_var('ARCH');
+        my $os_version = script_output('sed -rn "s/^VERSION_ID=\"(.*)\"/\1/p" /etc/os-release');
+        assert_script_run "SUSEConnect -p sle-module-legacy/$os_version/$arch";
```

Comment:
Does SUSEConnect exits with RC=0 if Legacy module is already installed? (didn't try myself)

Reply:
@ldevulder Yes.